### ISSUE TITLE
how to set s and m?

### ISSUE THREAD

**Issue by onlinehuazai** [ronghuaiyang/arcface-pytorch#90]
the acc is 0, triplet loss and CE loss can obtain 0.5 acc, is hyper-parameters(s and m)?

**Comment by Piaoyyt** [comment]
In the trainig progress, the trainig loss is decreasing but the accuracy on the training dataset is always zero. I don't know why this happens?

**Comment by crx2000** [comment]
in arcface paper, the author set s=64,m=0.5, but i don't know it is correct  to this project
